src/apps/core/submodules/alerts/alerts.scss

```diff
@@ -0,0 +1,157 @@
+@import "../../../../css/partials/base";
+
+#main_topbar_alerts {
+	#main_topbar_alerts_container {
+		// Transition to hide
+		visibility: hidden;
+		opacity: 0;
+		-webkit-transition: visibility 0.25s, opacity 0.25s ease-in-out;
+		   -moz-transition: visibility 0.25s, opacity 0.25s ease-in-out;
+			-ms-transition: visibility 0.25s, opacity 0.25s ease-in-out;
+			 -o-transition: visibility 0.25s, opacity 0.25s ease-in-out;
+				transition: visibility 0.25s, opacity 0.25s ease-in-out;
+
+		// Rest of styles
+		width: 425px;
+		margin-top: 8px;
+		margin-left:-356px;
+		background-color: $woodsmoke;
+		font-size: 14px;
+
+		&::before {
+			display: block;
+			float: right;
+			width: 1px;
+			margin-top: -5px;
+			margin-right: 45px;
+			border-left: 5px solid transparent;
+			border-right: 5px solid transparent;
+			border-bottom: 5px solid $woodsmoke;
+			content: '';
+		}
+
+		.dropdown-header {
+ 	 		box-sizing: border-box;
+			height: 36px;
+			border-radius: 2px 2px 0 0;
+			box-shadow: 0 1px 0 0 $midgray2;
+			color: $french-gray2;
+			padding: 12px 16px;
+			font-weight: 600;
+			line-height: 14px;
+		}
+
+		.dropdown-body {
+			color: $manatee;
+			overflow-y: auto;
+
+			.alert-group{
+				&:last-child .alert-item  {
+					border-radius: 0 0 2px 2px;
+				}
+
+				.alert-header {
+					padding: 4px 16px;
+					font-size: 12px;
+					font-weight: 600;
+					letter-spacing: 0.4px;
+					text-transform: uppercase;
+				}
+
+				.alert-item {
+					background: $shark2;
+					box-shadow: inset 0 -1px 0 0 $woodsmoke,
+					inset 0 1px 0 0 $woodsmoke;
+					padding: 12px 16px;
+
+					* {
+						margin-left: 48px;
+						margin-right: 0;
+						margin-bottom: 0;
+						color: $white;
+						font-size: 12px;
+						font-weight: normal;
+						line-height: 12px;
+					}
+
+					.icon {
+						float: left;
```

Comment:
Have you heard about this cool thing called `flexbox` ;)

Try using instead of floating elements. You might need to add an extra container that will contain the content of your alert. Again, better semantic structure with clear class name that are self explanatory in that context is what we aim for here.